testgrid/config/config.yaml

```diff
@@ -1038,6 +1038,8 @@ dashboards:
     test_group_name: kubernetes-e2e-gce-slow-release-1.5
   - name: gce-ha-master
     test_group_name: kubernetes-e2e-gce-ha-master
+  - name: gce-taint-evict
+    test_group_name: kubernetes-e2e-gce-taint-evict
```

Comment:
Need to define the gcs entry from [here](https://github.com/gmarek/test-infra/blob/master/testgrid/config/config.yaml#L861-L862)